### PR TITLE
Adding ASHeaderService CSC, HeaderService for the All Sky Camera

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

**Comment by mareuter** [comment]
The main check failed since one of the unit test files was not updated. Please see the _Test Utilities_ section of the README in https://github.com/lsst-ts/ts_xml.

**Comment by menanteau** [comment]
Added ASHeaderService to testutils.py

**Review by tribeiro — CHANGES_REQUESTED** [review]
Sorry @menanteau , as we've been discussing in slack. I would like to propose we rename this CSC GCHeaderService (GC - Generic Camera) and make it an indexed component to match that of the GenericCamera.

**Comment by menanteau** [comment]
@mareuter  and @tribeiro I have renamed ASHeaderService to GCHeaderService as discussed.

File: sal_interfaces/SALSubsystems.xml [hunk 1]
```diff
@@ -361,6 +361,24 @@
     <Simulator>Not Required</Simulator>
     <Configuration>Not Configurable</Configuration>
   </SALSubsystem>
+  <SALSubsystem>
+    <Name>GCHeaderService</Name>
+    <Description>Indexed HeaderService for Generic Camera</Description>
+    <IndexEnumeration>no</IndexEnumeration>
```
Comment (tribeiro) [inline sal_interfaces/SALSubsystems.xml:367]: This need to be:

```
<IndexEnumeration>any</IndexEnumeration>
```

Comment (menanteau) [reply sal_interfaces/SALSubsystems.xml]: Fixed

Comment (tribeiro) [reply sal_interfaces/SALSubsystems.xml]: thank you so much!

**Review by tribeiro — APPROVED** [review]
Thanks for the updates!

**Review by mareuter — APPROVED** [review]
This is good to merge.

**Pull request merged**